### ISSUE TITLE
why this request execute two times

### ISSUE THREAD

**Issue by javinjiao** [1iveowl/Simple-Http-Listener-PCL#37]
https://github.com/1iveowl/Simple-Http-Listener-PCL/blob/e272ffa96bf802e30410ba101b9a8980af8492ad/src/test/Console.NETCore.Test/Program.cs#L52
I just send one request and each time here has execute two times.

**Comment by 1iveowl** [comment]
I moved all future development to SimpleHttpListener.Rx. I did a lot of improvements to that version.

Could you look at that instead, please. I believe it have solved this issue too.

**Comment by javinjiao** [comment]
Yeah, but after retry [SimpleHttpListener.Rx](https://github.com/1iveowl/SimpleHttpListener.Rx),this issue still arise.

**Comment by 1iveowl** [comment]
OK. I'll have a look at it with the Rx version. Would you mind sharing some more details for how I reproduce the issue and preferably move the issue to the Rx repo?
